yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -427,14 +446,35 @@ func (yb *TargetYugabyteDB) InitEventChannelsMetaInfo(migrationUUID uuid.UUID, n
 			return false, nil
```

Comment:
If channel info is created but not the stats table entries, this `return` statement is undesired! You should move the "check if channel metadata is already initialised" and "init channel metadata" into a separate function named `initChannelMetadata()`.